contracts/interfaces/InterestConsensusInterface.sol

```diff
@@ -14,10 +14,8 @@ interface InterestConsensusInterface {
 
     event InterestAccepted(address lender, uint256 blockNumber, uint256 interest);
 
-    function submitInterestResult(
-        ZeroCollateralCommon.Signature calldata signature,
-        address lender,
-        uint256 blockNumber,
-        uint256 interest
-    ) external;
+    function processRequest(
```

Comment:
I think signer and lender should be indexed in the InterestSubmitted event.

Reply:
done